lib/rules/v-if-else-key.js

```diff
@@ -44,35 +67,36 @@ const checkForKey = (
   uniqueKey,
   conditionalFamilies
 ) => {
-  if (node.parent && node.parent.type === 'VElement') {
+  if (
+    node.parent &&
+    node.parent.type === 'VElement' &&
+    hasConditionalRenderedSiblings(node, componentName)
+  ) {
     const conditionalFamily = conditionalFamilies.get(node.parent)
 
-    if (
-      conditionalFamily &&
-      (utils.hasDirective(node, 'bind', 'key') ||
-        utils.hasAttribute(node, 'key') ||
-        !hasConditionalDirective(node) ||
-        !(conditionalFamily.else || conditionalFamily.elseIf.length > 0))
-    ) {
-      return
-    }
+    if (conditionalFamily && !utils.hasAttribute(node, 'key')) {
```

Comment:
Can you rewrite this into an early return again please?

Reply:
[f9600d5](https://github.com/vuejs/eslint-plugin-vue/pull/2348/commits/f9600d5aa2714a9a2a6dd9621bbbd7d97e0af44d)